packages/docusaurus-theme-classic/src/theme/IconHome/index.tsx

```diff
@@ -8,15 +8,9 @@
 import React from 'react';
 import type {Props} from '@theme/IconHome';
 
-export default function IconDarkMode(props: Props): JSX.Element {
+export default function IconHome(props: Props): JSX.Element {
   return (
-    <svg
-      xmlns="http://www.w3.org/2000/svg"
-      height="24"
-      width="24"
-      viewBox="0 0 24 24"
-      {...props}>
-      <path d="M0 0h24v24H0V0z" fill="none" />
```

Comment:
Interesting, I vividly remember I removed this... Must have forgotten to push it, then